iris/publish-model.conf

```diff
@@ -438,6 +438,32 @@ publish {
       ]
     }
 
+    {
```

Comment:
Good that you put this back in. One of the main reasons this value was published by the TCS in the first place is so that this angle is available to the RPG (AOESW) so that it can get all of the coordinate transformation done correctly. I've added @lianqiw  as a reviewer to confirm.